src/main/kotlin/com/exactpro/th2/codec/json/TypeInfoDeserializer.kt

```diff
@@ -18,23 +18,22 @@ package com.exactpro.th2.codec.json
 import com.fasterxml.jackson.core.JsonParser
 import com.fasterxml.jackson.core.JsonTokenId
 import com.fasterxml.jackson.databind.DeserializationContext
+import com.fasterxml.jackson.databind.JavaType
 import com.fasterxml.jackson.databind.deser.std.UntypedObjectDeserializer
 import java.io.IOException
 import java.math.BigDecimal
 
 
-class TypeInfoDeserializer : UntypedObjectDeserializer() {
+class TypeInfoDeserializer(listType: JavaType, mapType: JavaType) : UntypedObjectDeserializer(listType, mapType) {
```

Comment:
> UntypedObjectDeserializer(listType, mapType)

According to the deprecated constructor passing `null`s would be sufficient too, but okay